snappi_ixnetwork/trafficitem.py

```diff
@@ -1366,3 +1368,97 @@ def _construct_timestamp(self, flow_row, row):
                 )
         if len(timestamp_result) > 0:
             flow_row["timestamps"] = timestamp_result
+
+    def update_flows(self, update_flows_config):
+        """
+        Update the flows with property size & rate
+        """
+        self._validate_update_flows_config(update_flows_config)
+        for flow in update_flows_config.flows:
+            hl = self._api._ixnetwork.Traffic.TrafficItem.find(
+                Name=flow.name
+            ).HighLevelStream.find()
+            self._update_size(hl, flow.size)
+            self._update_rate(hl, flow.rate)
+
+    def _validate_update_flows_config(self, update_flows_config):
+        errors = []
+        for flow in update_flows_config.flows:
+            if flow not in self._api._config.flows._items:
```

Comment:
We may reduced multiple search call by passing all name of traffic items. It will enhance some performance.

Reply:
As discussed, we might get different combinations of update rate or size per different flows. Won't be a feasible solution to get all traffic items and apply rate or size per item